runscope/resource_runscope_environment.go

```diff
@@ -83,6 +83,51 @@ func resourceRunscopeEnvironment() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+			"webhooks": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"emails": &schema.Schema{
+				Type: schema.TypeList,
```

Comment:
Is the order of the emails important? `TypeList` should only be used for ordered lists, otherwise you should use a `TypeSet`. The reason is that ordering changes can result in spurious diffs